src/meshes/t8code_mesh.jl

```diff
@@ -37,7 +39,9 @@ mutable struct T8codeMesh{NDIMS, RealT <: Real, IsParallel, NDIMSP2, NNODES} <:
         mesh.nodes = nodes
         mesh.boundary_names = boundary_names
         mesh.current_filename = current_filename
+        mesh.unsaved_changes = true # TODO
         mesh.tree_node_coordinates = tree_node_coordinates
+        mesh.max_number_faces = 4 # TODO: How to automatically adapt for other element types, e.g. triangles?
```

Comment:
Is there a t8code routine that gives the maximum number of faces per element in a forest?

Reply:
Outdated

Reply:
Outdated